app/services/shopify_import/variants/data_parsers/base_data.rb

```diff
@@ -0,0 +1,49 @@
+module ShopifyImport
+  module Variants
+    module DataParsers
+      class BaseData
+        def initialize(shopify_variant, spree_product)
+          @shopify_variant = shopify_variant
+          @spree_product = spree_product
+        end
+
+        def variant_attributes
+          {
+            sku: @shopify_variant.sku,
```

Comment:
How do you resolve the case when there's no SKU? I remember we talked that in Shopify variant SKU is not required, while in spree it is. I think the easiest solution would be to "generate" the SKU. That will cause the least number of issues. The other solution would be to skip validation, but I'm afraid at how many bugs that can produce later (for example when admin tries to update the variant, or any other extension tries to make some change)